packages/website/src/hooks/useDebouncedToggle.ts

```diff
@@ -1,6 +1,6 @@
 import { useCallback, useRef, useState } from 'react';
 
-export default function useDebouncedToggle<T>(
+export function useDebouncedToggle<T>(
   value: T,
   timeout = 1000,
```

Comment:
[Accessibility] Ah, I missed this before, sorry - it's an accessibility issue to have parts of the page that exist for only a limited time. If someone isn't able to read the _Copied_ message in time, such as from a reading disability or missing it on the page, they could miss what it said and not know whether the copy was successful.

Since this is existing code I'm still happy to ✔️ and file a followup issue - just raising an FYI.

FWIW one remediation I've liked in the past is to make the copy button a ✔️ until the value that would be copied is changed. The _Copied_ text can then disappear as long as there's _some_ indication of the (success) status. 

https://www.w3.org/WAI/WCAG21/Understanding/enough-time

Reply:
this is oo, scope for this change, we should do follow-up to improve accessibility